packages/electrode-archetype-react-app-dev/lib/webpack-dev-hapi.js

```diff
@@ -51,10 +51,9 @@ function register(server, options, next) {
   } else {
     config.entry = hmrClient.concat(config.entry);
   }
-
+  
   config.plugins = [
-    new Webpack.HotModuleReplacementPlugin(),
-    new Webpack.NoEmitOnErrorsPlugin()
+    new Webpack.HotModuleReplacementPlugin()
```

Comment:
do we need to add the optimization.noEmitOnErrors here?

Reply:
updated @datvong-wm